content/rc/databases/migrate-databases.md

```diff
@@ -55,9 +57,11 @@ An error will appear when syncing the two databases if the source database has [
 An error will appear when syncing the two databases if the source and target databases are hosted on different Redis Cloud accounts. [Contact support](https://redis.com/company/support/) if you want to migrate a database between accounts using Active-Passive.
 {{< /note >}}
 
-Here's how this works for databases hosted on the same account:
+### Detailed process for Active Passive syncing: databases on same account
```

Comment:
This header is a bit long. Preferably it should fit on a single line.
```suggestion
### Detailed Active-Passive syncing process
```
OR
```suggestion
### Detailed Active-Passive syncing
```
OR
```suggestion
### Sync databases on same account
```